common/flatpak-dir.c

```diff
@@ -12993,7 +13019,7 @@ flatpak_dir_get_default_locale_languages (FlatpakDir *self)
 char **
 flatpak_dir_get_locale_languages (FlatpakDir *self)
 {
-  GKeyFile *config = ostree_repo_get_config (self->repo);
+  GKeyFile *config = flatpak_dir_get_repo_config (self);
```

Comment:
This one also needs to work if ensure_repo fails, so one can get the current langs.

The other ones are ok as is, as they all take a remote name as argument, which implies that there is a valid system repo.

Reply:
I believe it does - if config is NULL we return the default value.

I'll add tests to for this at some point